geoopt/manifolds/lorentz/math.py

```diff
@@ -0,0 +1,607 @@
+import torch.jit
+import torch as th
+
+MIN_NORM = 1e-15
+
+
+class Arcosh(torch.autograd.Function):
```

Comment:
I no see little point in doing this. We could use torch script everywhere. It is well supported since 1.3.1. see K-stereographic for reference.